gallery/src/data-visualization/ReactVisBarGraphPage.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { GalleryDescriptionTile, GalleryExampleTile } from '../gallery-components/GalleryTiles';
+import ReactVisBarGraphExample from './examples/ReactVisBarGraphExample';
+import { NxTable, NxTableBody, NxTableCell, NxTableHead, NxTableRow } from '@sonatype/react-shared-components';
+
+const reactVisBarGraphExampleCode = require('./examples/ReactVisBarGraphExample?raw');
+
+const ReactVisBarGraphPage = () =>
+  <>
+    <GalleryDescriptionTile >
+      <p className="nx-p">
```

Comment:
Don't forget we have the convenience components which make a lot of these simpler

Reply:
> Don't forget we have the convenience components which make a lot of these simpler

I have changed the code to use convenience components where required.
2e4db1dfde848ef990f90db8c18ece20a6268f2b